### DISCUSSION TITLE
Downloading UGA Feed gets “download_latest_feed_version”

### DISCUSSION THREAD

**Discussion by ABonney8** [transitland/transitland#425] [Q&A] [answered]
I’ve been attempting to use TransitLand to obtain the University of Georgia’s current GTFS feeds but am having trouble downloading it. I’m signed into my account and I’m able to click the download button, but it seems it just downloads a file called “download_latest_feed_version” that I can’t use. 

The exact GTFS file I’m looking to download is linked here: (https://www.transit.land/feeds/f-uga~ga~us/versions/25a70bf6cd69ca229db221c467629e58c87135eb)

Any help would be greatly appreciated!

**Comment by drewda** [comment, marked as answer]
@ABonney8 Try renaming the file and add a `.zip` extension to the end of it.

We can look into improving this behavior on the server-side.
